packages/roosterjs-editor-core/lib/corePlugins/TypeInContainerPlugin.ts

```diff
@@ -30,6 +30,13 @@ export default class TypeInContainerPlugin implements EditorPlugin {
         this.editor = null;
     }
 
+    isRangeEmpty(range: Range) {
+        if (range.collapsed && !range.startContainer.firstChild) {
```

Comment:
If startContainer is text node, it will always return true

Better check if it is an element, and check the tag name is DIV